src/sentry/reprocessing2.py

```diff
@@ -232,8 +237,12 @@ def start_group_reprocessing(project_id, group_id, max_events=None, acting_user_
         new_group.times_seen = 0
         new_group.save()
 
-        for model in GROUP_MODELS_TO_MIGRATE:
-            model.objects.filter(group_id=group_id).update(group_id=new_group.id)
+    # This migrates all models that are associated with a group but not
+    # directly with an event, i.e. everything but event attachments and user
+    # reports. Those other updates are run per-event (in
+    # post-process-forwarder) to not cause too much load on pg.
+    for model in GROUP_MODELS_TO_MIGRATE:
+        model.objects.filter(group_id=group_id).update(group_id=new_group.id)
```

Comment:
Wasn't the point of `DIRECT_GROUP_RELATED_MODELS` to be able to do this in an atomic transaction?

Reply:
not sure why I moved this out. We'll have to see if it causes problems